src/main/scala/beam/router/BeamRouter.scala

```diff
@@ -5,24 +5,13 @@ import java.util.UUID
 import java.util.concurrent.TimeUnit
 
 import akka.actor.Status.{Status, Success}
-import akka.actor.{
-  Actor,
-  ActorLogging,
-  ActorRef,
-  Address,
-  Cancellable,
-  ExtendedActorSystem,
-  Props,
-  RelativeActorPath,
-  RootActorPath,
-  Stash
-}
+import akka.actor.{Actor, ActorLogging, ActorRef, Address, Cancellable, ExtendedActorSystem, Props, RelativeActorPath, RootActorPath, Stash}
```

Comment:
Did you run scalafmt? If this passes that then I'll be amazed :) - Just pre-empting the failure in CI

Reply:
Nope, just have seen that it failed. Fixed and force pushed once again. Thanks.